Move External Identities APIs to v1.0 (#12210)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/resources/userattributevaluesitem.md

---
title: "userAttributeValuesItem resource type"
description: "Used to populate the values for a user flow attribute within a user flow when there are multiple selections to choose from."
author: "jkdouglas"
localization_priority: Normal
ms.prod: "identity-and-sign-in"
doc_type: resourcePageType
---

# userAttributeValuesItem resource type

Namespace: microsoft.graph

> Used to populate the values for a user flow attribute within a user flow when there are multiple selections to choose from. userAttributeValuesItem is applicable to the userInputTypes `radioSingleSelect`, `dropdownSingleSelect`, and `checkboxMultiSelect` for an [identityUserFlowAttributeAssignment](..\resources\identityuserflowattributeassignment.md).

Reviewer: ```suggestion
Represents user flow attribute values within a user flow when there are multiple selections to choose from.  The userAttributeValuesItem is applicable to the userInputTypes of `radioSingleSelect`, `dropdownSingleSelect`, and `checkboxMultiSelect` for an [identityUserFlowAttributeAssignment](..\resources\identityuserflowattributeassignment.md).
```

## Properties

|Property|Type|Description|
|:---|:---|:---|

> |isDefault|Boolean|Used to set the value as the default.|

Reviewer: ```suggestion
|isDefault|Boolean|Determines whether the value is set as the default.|
```
